Dockerfile

```diff
@@ -0,0 +1,48 @@
+ARG DOCKER_IMAGE=fedora:latest
+FROM ${DOCKER_IMAGE}
+
+WORKDIR /build
+COPY . .
+
+ENV PY_COLORS=1
+
+RUN echo -e "deltarpm=0\ninstall_weak_deps=0\ntsflags=nodocs" >> /etc/dnf/dnf.conf
+
+# Temporary workaround until fedora:rawhide image is updated with up-to-date
+# fedora-repos package.
+RUN curl -O https://pagure.io/fedora-repos/raw/master/f/RPM-GPG-KEY-fedora-28-primary
+RUN mv RPM-GPG-KEY-fedora-28-primary /etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-28-x86_64
+RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-28-x86_64
+
+RUN dnf -y update
+RUN dnf -y install yum
```

Comment:
Fixed mock package is now in both F25 and F26, so you can remove this workaround.